workshops/models.py

```diff
@@ -227,6 +227,16 @@ def unpublished_events(self):
 
         return self.filter(published=False)
 
+    def unpaid_events(self):
+        '''Return a queryset for events that owe money.'''
+
+        # All events that have a non-zero admin fee.
+        queryset = self.filter(admin_fee__ne=0)
+
+        # Of those, all those whose paid status is not 'Yes' (i.e., is 'No' or NULL).
+        queryset = queryset.filter(fee_paid__ne=True)
```

Comment:
Not sure if it's possible to store negative numbers in DecimalField, but I thought a little and it makes no sense to list overpaid events under "unpaid" label.

And I like the chained calls.